providers/src/airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -157,7 +157,11 @@ class KubernetesPodOperator(BaseOperator):
     :param labels: labels to apply to the Pod. (templated)
     :param startup_timeout_seconds: timeout in seconds to startup the pod.
     :param startup_check_interval_seconds: interval in seconds to check if the pod has already started
+    :param get_init_containers_logs: get the stdout of the init containers as logs of the tasks.
     :param get_logs: get the stdout of the base container as logs of the tasks.
+    :param init_container_logs: list of init containers whose logs will be published to stdout
+        Takes a sequence of containers, a single container name or True. If True,
+        all the containers logs are published. Works in conjunction with get_init_containers_logs param.
```

Comment:
Is this parameter not partly redundant with the other bool parameter?
I would assume if there are mutliple init containers I can use this sequence to limit from whoich init containers I want to have logs. But if I specify a list I don't need another bool.
On the other hand if I just want to have all init container logs, then the list need to be filled?

Reply:
Yes, you're right; we don't need get_init_containers_logs because for the main containers, we log the base container, whereas for init containers, we don't use this term. 

So, I'd suggest using init_container_logs instead.